File: 1761A-Permutation.py
t_str = input()
if t_str:
    t = int(t_str)
    
    for _ in range(t):
        n, a, b = map(int, input().split())

        x = (n == a == b)
        y = (a + b <= n - 2)
        
        if x or y:
            print("Yes")
        else:
            print("No")

File: 1761A-Permutation.cpp
#include <iostream>
using namespace std;

int main() {
    int t;

    if (!(cin >> t)) return 0;

    while (t--) {
        int n, a, b;
        cin >> n >> a >> b;

        bool x = (n == a && a == b);
        bool y = (a + b <= n - 2);

        if (x || y) {
            cout << "Yes" << endl;
        } else {
            cout << "No" << endl;
        }
    }
    return 0;
}
